packages/postcss-normalize-unicode/src/__tests__/index.js

```diff
@@ -12,59 +12,68 @@ test(
     'should convert a unicode range to a wildcard range',
     processCSS,
     fixture('u+2b00-2bff'), // Miscellaneous Symbols and Arrows
-    fixture('u+2b??')
+    fixture('u+2b??'),
+    {env: 'chrome58'}
```

Comment:
ie11 is in the list of browsersList by "defaults" selector.  http://browserl.ist/
So that means '+u' will be upper cased and standart test will fail.

Reply:
@vitaliymaz hm, okey, let's add test where ie11 in not browserslist

Reply:
Do you mean use something like {env: 'not ie'} instead of {env: 'chrome58'}?

Reply:
@vitaliymaz any